content/blog/2023/09/kwctl-sha.md

```diff
@@ -0,0 +1,35 @@
+---
+title: kwctl SHA support
+authors:
+  - Fabrizio Sestito
+date: 2023-09-06
+---
+
+Lately, we have been focusing on improving the developer experience of Kubewarden, and in general,
+we are striving to implement features that are being requested by the community.
+
+# Reference policies by their SHA
+
+Today we are happy to announce that we have released a new version of `kwctl` that supports
+referencing policies by their SHA.
+
+Container engines such as Docker and Podman allow users to refer to images by their SHA sum.
+
+Being Kubewarden policies distributed as OCI artifacts, we tought it would be a good idea
+to add the SHA support to `kwctl`, so that the user could find a familiar experience.
```

Comment:
```suggestion
to add the SHA support to `kwctl`, so that the user has a familiar experience.
```

(usually, having less passive voices is easier for reading)